R/simulation.R

```diff
@@ -82,7 +84,14 @@ Simulation <- R6::R6Class(
       mw <- rClr::clrCall(self$ref, "MolWeightFor", quantityPath)
       mw %||% NA
     },
-
+    #' @description
+    #' Returns the applications ordered by start time associated to the quantity with path `quantityPath` or an emtpty list if not found
+    #' @param quantityPath Path of quantity used to retrieve the applications (e.g. applications resulting in this quantity being applied)
+    allApplicationsFor = function(quantityPath) {

```

Comment:
This is how you will get all the application for a given quantity

I do not know if using `quantityPath` makes sense here of if it should be `moleculeName`

This is for the moment using the same approach as for `MoleculeWeight` .Let me know what makes more sense